science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+<thead>
+<tr>
+<th style="text-align:left;">
+Release
+</th>
+<th style="text-align:left;">
+In der CWA implementierte Funktionen
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.0
+</td>
+<td style="text-align:left;width: 11cm; ">
+Kernfunktionalitäten (Proximity-Tracing via Bluetooth LE, Teilen positiver Corona-Testergebnisse, Warnung anderer Nutzender), Zählung aktiver Tage
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.1
+</td>
+<td style="text-align:left;width: 11cm; ">
+Textanpassungen, Verbesserung der Barrierefreiheit, Übersetzung: Türkisch
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.2
+</td>
+<td style="text-align:left;width: 11cm; ">
+Zulassen von Screenshots, Übersetzung: Bulgarisch, Polnisch, Rumänisch
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.3
+</td>
+<td style="text-align:left;width: 11cm; ">
+Zusätzliche Informationen zum Risikostatus, Textverbesserung bei angezeigtem niedrigen Infektionsrisiko, Korrektur der Rechtstexte
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.5
+</td>
+<td style="text-align:left;width: 11cm; ">
+Unterstützung des europäischen Corona-App-Gateways, freiwillige Symptomerfassung, Verbesserung der Textverständlichkeit
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.6
+</td>
+<td style="text-align:left;width: 11cm; ">
+Detaillierte Erklärung bei Wechsel des Risikostatus
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.7
+</td>
+<td style="text-align:left;width: 11cm; ">
+Risikoüberprüfung kann mehrmals täglich durchgeführt werden, Erinnerungsfunktion für die Teilung positiver Testergebnisse via CWA
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.9
+</td>
+<td style="text-align:left;width: 11cm; ">
+Umstellung auf G/A-ENF Version 2, Verbesserung der Risikobewertung, Verbesserung vom Ablauf des Einscannens der Testergebnisse bis zum Teilen der Diagnoseschlüssel
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.10
+</td>
+<td style="text-align:left;width: 11cm; ">
+Kontakttagebuch
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.11
+</td>
+<td style="text-align:left;width: 11cm; ">
+Kennzahlen zum Infektionsgeschehen
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.12
+</td>
+<td style="text-align:left;width: 11cm; ">
+Erweiterung des Kontakttagebuchs (Begegnungshistorie), Unterstützung von den iPhones 5s, 6 und 6 Plus
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.13
+</td>
+<td style="text-align:left;width: 11cm; ">
+Freiwillige Datenspende (Privacy-Preserving-Analytics (PPA))
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.14
+</td>
+<td style="text-align:left;width: 11cm; ">
+Erweiterung des Kontakttagebuchs (u.a. Pop-Up-Menü), Erweiterung der freiwilligen Datenspende (neue Datenpunkte)
```

Comment:
"Kontakttagebuchs" => "Kontakt-Tagebuchs"

Reply:
Fixed in the latest commit, thanks!